<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Модули</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="decksList"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="learnDeckButton">
        <property name="text">
         <string>Изучить модуль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="repeatDeckButton">
        <property name="text">
         <string>Повторить модуль</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addDeckButton">
        <property name="text">
         <string>Добавить модуль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editDeckButton">
        <property name="text">
         <string>Редактировать модуль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteDeckButton">
        <property name="text">
         <string>Удалить модуль</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Коллекция</string>
    </property>
    <addaction name="importAction"/>
    <addaction name="exportAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="importAction">
   <property name="text">
    <string>Импортировать</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>Экспортировать</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
